charts/drax/values.yaml

```
@@ -645,6 +649,18 @@ grafana:
             organization: accelleran
             defaultBucket: default
             tlsSkipVerify: true
+        - name: InfluxDB2
+          type: influxdb
+          access: proxy
+          url: "http://{{ $.Release.Name }}-influxdb2.{{ $.Release.Namespace }}:80"
+          uid: GHBC3g9KVb
```

Let's take something properly random iso just changing a couple of characters:

```suggestion
          uid: ddswjfzs92bk0b
```

---

OK - this was just done quickly in out interactive session